src/Utilities/universalfallback.jl

```diff
@@ -13,7 +13,7 @@ optimizer bridges should be used instead.
 """
 mutable struct UniversalFallback{MT} <: MOI.ModelLike
     model::MT
-    constraints::Dict{Tuple{DataType, DataType}, Dict} # See https://github.com/JuliaOpt/JuMP.jl/issues/1152
+    constraints::OrderedDict{Tuple{DataType, DataType}, Dict} # See https://github.com/JuliaOpt/JuMP.jl/issues/1152 and https://github.com/JuliaOpt/JuMP.jl/issues/2238
```

Comment:
Let's have a think about this type. Do we gain anything from `Tuple{DataType, DataType}` and `Dict`? Why not just `OrderedDict{Any, Any}`? I can't imagine the compiler can specialize.

Reply:
At least it helps in readability. What would be the advantage of using `Any`?